contracts/Octobay.sol

```diff
@@ -360,6 +366,59 @@ contract Octobay is Ownable, ChainlinkClient, BaseRelayRecipient {
 
 
 
+    // ------------ GOVERNANCE ------------ //
+
+
+    OctobayGovernor public octobayGovernor;
+    OctobayGovTokenFactory public octobayGovTokenFactory;
+
+    struct NewGovernanceToken {
+        bool isValue;
```

Comment:
I get the point of this. Was just wondering if there's a nicer solution. I thought about storing the `requestId` there but a bool value is better I guess. The thing is, it wasn't very clear to me at first what this does, so I'd maybe just rename it to something like `hasBeenRequested` or `pending`.

Reply:
I add this to any struct I use in a map actually. As there's no good way in Solidity to know if you have a 'real' value in a map or not, as every possible key maps to 'zero values' of the struct essentially. Sometimes it's enough to know a 'zero value' is not 'real', but that's not always the case. The bool makes things clearer to me.